packages/timeline/src/ReactAxisGanttSeries.ts

```diff
@@ -0,0 +1,255 @@
+import { min as d3Min, max as d3Max } from "d3-array";
```

Comment:
You can also get them from `@hpcc-js/common` (but the rollup config aliases them anyway).